README.md

```diff
@@ -91,6 +91,55 @@ Documentation from the google-java-format CLI tool :
   Do not remove unused imports. Imports will still be sorted.
 ```
 
+### Advanced profile
+If you wish to install the pre-commit hook only if it is missing, you can use the following profile :
+```xml
+<profiles>
+  <profile>
+    <!-- Install pre-commit hook if missing-->
+    <id>install-pre-commit-hook</id>
+    <activation>
+      <file>
+        <missing>.git/hooks/pre-commit</missing>
```

Comment:
The pre-commit file may have been added by another system. In this case, this library pre commit hook would never install.
I think this section should be removed since it is not applicable to all situations.

Reply:
will remove